File: lista2sala/5.py
a=float(input("digite o 1º valor:"))
b=float(input("digite o 2º valor:"))
c=float(input("digite o 3º valor:"))
if (a<b) and (a<c) and (b<c):
    print("os valores em ordem crescente são:", a, b, c)
elif a<b and a<c and c<b:
    print("os valores em ordem crescente são:", a, c, b)
elif b<a and b<c and a<c:
    print("os valores em ordem crescente são:", b, a, c)
elif b<a and b<c and c<a:
    print("os valores em ordem crescente são:", b, c, a)
elif c<a and c<b and a<b:
    print("os valores em ordem crescente são:", c, a, b)
else:
    print("os valores em ordem crescente são:", c, b, a)
            
    

File: listas/lista2sala/5.c
#include <stdio.h>
#include <stdlib.h>
int compare_ints(const void *a, const void *b) {
    return (*(int*)a - *(int*)b);
}

int main() {
    int values[3];

    for (int i = 0; i < 3; i++) {
        printf("Digite o valor %d: ", i + 1);
        scanf("%d", &values[i]);
    }
    qsort(values, 3, sizeof(int), compare_ints);

    printf("Valores em ordem crescente: %d %d %d\n", values[0], values[1], values[2]);

    return 0;
}
